standup/api/views.py

```diff
@@ -190,3 +190,50 @@ def post(self, request):
         status.save()
 
         return HttpResponseJSON({'id': status.id, 'content': content})
+
+
+class StatusDelete(AuthenticatedAPIView):
+    def delete(self, request, pk):
+        token = request.auth_token
+
+        # FIXME: Authorize this operation.
+
+        username = request.json_body.get('user')
+
+        if not username:
+            return HttpResponseBadRequest('Missing required fields.')
+
+        status = Status.objects.filter(id=pk).first()
+        if not status:
+            return HttpResponseBadRequest('Status does not exist.')
+
+        if status.user.username != username:
+            return HttpResponseForbidden('You cannot delete this status.')
+
+        status_id = status.id
+        status.delete()
+
+        return HttpResponseJSON({'id': status_id})
+
+
+class UpdateUser(AuthenticatedAPIView):
+    def post(self, request, username):
+        token = request.auth_token
+
+        # FIXME: Authorize this operation.
+
+        user = StandupUser.objects.filter(user__username=username).first()
```

Comment:
Mmm... So, the old code for the API is here:

https://github.com/mozilla/standup/blob/36120b10d6906d174f414e6766272ba25c3386bd/standup/apps/api/views.py#L218

I think I want to leave this as is because that's how it "works" now, but we should fix it when we work on that "wtf irc nick, slug, username, ..." bug.